index.js

```diff
@@ -72,8 +72,9 @@ function assembleStyles() {
 		}
 	};
 
-	// Fix humans
-	styles.color.grey = styles.color.gray;
+	// Alias bright black as gray (and grey)
+	styles.color.grey = styles.color.gray = styles.color.brightBlack;
+	styles.bgColor.bgGrey = styles.bgColor.bgGray = styles.bgColor.bgBrightBlack;
```

Comment:
Code style forbids chained assignment - could you break them out onto their own lines?

Reply:
updated